docs/content/any/project/changelogs/2021-09-29.md

```diff
@@ -15,37 +15,51 @@ description: >-
 - Unification constraints on dot properties are no longer elided when emitting 
   partial results for queries over multiple unbound variables.
 
-### JavaScript
-
-#### Other bugs & improvements
-- Data filtering API functions (`authorizedQuery`, `authorizedResources`) now
-  correctly restrict their output to records of the user-supplied resource type.
-
-
-### Go
+### Node.js
 
 #### Breaking changes
 
-<!-- TODO: remove warning and replace with "None" if no breaking changes. -->
-
 {{% callout "Warning" "orange" %}}
   This release contains breaking changes. Be sure to follow migration steps
   before upgrading.
 {{% /callout %}}
 
-##### Breaking change 1
+##### Undefined attributes now error
+
+To help with catching erroneous policies, accessing an undefined attribute
+on an object will now return an error.
 
-Summary of breaking change.
+Policy:
 
-Link to [migration guide]().
+```polar
+allow(user, "read", "document") if user.isAdmin;
+```
 
-#### New features
+Code:
 
-##### Feature 1
+```js
+const user = {name: "notanadmin"};
+await oso.isAllowed(user, "read", "document");
+```
 
-Summary of user-facing changes.
+Before this would silently fail, since `user.isAdmin = undefined` in JavaScript, and
+`undefined` is not `true`.
 
-Link to [relevant documentation section]().
+Now, this will error since `isAdmin` is not a property on `user`.
+
+This will still work (1) if `user = {isAdmin: undefined}`,
+and (2) by instead using the policy
```

Comment:
```suggestion
This will still work if (1) the `isAdmin` property is explicitly set to `undefined`
(e.g., `user = {isAdmin: undefined}`) or (2) the property check is moved into the head
of the rule:
```